module/checker.go

```diff
@@ -14,6 +14,17 @@ import (
 	"github.com/lyft/protoc-gen-star"
 )
 
+var unknown = ""
+var httpHeaderName = "^:?[0-9a-zA-Z!#$%&'*+-.^_|~\x2C]+$"
+var httpHeaderValue = "^[ \t]*(?:[\x20-\x7E\u0080-\u00FF](?:[ \t]+[\x20-\x7E\u0080-\u00FF])?)*[ \t]*$"
```

Comment:
Is it possible to just treat tab and space as allowed chars and simplify this? Or we do miss something that way?

Reply:
fwiw Envoy's internal header value check does consider space and tab as allowed characters (https://github.com/envoyproxy/envoy/blob/5248a4fb7d4c2a3d1fa151f944d3a63f6b7a06cf/source/common/http/header_utility.cc#L146). It just whitelists the allowed characters (https://github.com/nghttp2/nghttp2/blob/bb519154fe62f7ff7e5eb7269e05043dec6d3682/lib/nghttp2_helper.c#L43). Could simplify and just make it match for now

Reply:
If we simplify it and just make it a whitelist or blacklist (with the advantage of @SecurityInsanity's feedback of having a blacklist), we could either have:
whitelist incl tabs and space: `"^[\t\x20-\x7E\u0080-\u00FF]*$"`
blacklist control characters except tab: `"^[^\x00-\x08\x0A-\x1F\x7F]"`